addons/website/tests/test_snippets.py

```diff
@@ -13,3 +16,11 @@ def test_01_empty_parents_autoremove(self):
 
     def test_02_default_shape_gets_palette_colors(self):
         self.start_tour("/?enable_editor=1", "default_shape_gets_palette_colors", login='admin')
+
+    def test_03_snippets_all_drag_and_drop(self):
+        with MockRequest(self.env, website=self.env['website'].browse(1)):
+            snippets_template = self.env['ir.ui.view'].render_public_asset('website.snippets')
+        html_template = html.fromstring(snippets_template)
+        data_snippet_els = html_template.xpath("//*[@class='o_panel' and not(contains(@class, 'd-none'))]//*[@data-snippet]")
+        snippets_names = ','.join([el.attrib['data-snippet'] for el in data_snippet_els])
+        self.start_tour("/?enable_editor=1&snippets_names=%s" % snippets_names, "snippets_all_drag_and_drop", login='admin', timeout=300)
```

Comment:
TODO: probably split snippets_name by batch of 10/15/20 (there is 69 on full runbot) and start 3/4/5/6 tests instead of everything in a big one -> it will allow to reduce the timeout set to 300
The expected time the test should take is between 150-180 seconds